vgm.sh

```diff
@@ -0,0 +1,35 @@
+package: VGM
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "4.3"
+source: https://github.com/alisw/vgm.git
```

Comment:
> Fixed ( Although what is the convention here? Many *.sh contain 
> package names in capital letters ).

The best practice would be that the repository name matches the package 
name as specified in the .sh (or viceversa). The rationale for that is 
that if you want to develop a package locally, you do not need any extra 
directory name for the git clone.

The cases where this is not the case are historical naming of the 
packages wrt their original repository (e.g. ROOT was already ROOT on 
CVMFS, while the repository is root because that’s what SFT used).

Anyways, I tried to make the policy more explicit in #336.